src/modules/resource/reslog.c

```diff
@@ -197,6 +197,43 @@ int reslog_sync (struct reslog *reslog)
     return 0;
 }
 
+/*  Truncate resource journal if needed to reslog->journal_max
```

Comment:
commit message "replace the most recent event" ... perhaps you mean "replace the oldest event"?

Reply:
Meant to be "replace the most recently dropped event" - not the oldest event in case more than one event is dropped at a time.